### ISSUE TITLE
v3.7.7: TraditionalBibliographyPlugin does not implement `on_insert_citation`? 

### ISSUE THREAD

**Issue by xiaoweiz** [SublimeText/LaTeXTools#699]
Just updated to v3.7.7. The cite autocomplete is working but the console shows the following error message. 

```python
TraditionalBibliographyPlugin does not implement `on_insert_citation`
Exception in thread Thread-83:
Traceback (most recent call last):
  File "/Users/xiaoweiz/Library/Application Support/Sublime Text 3/Packages/LaTeXTools/latex_cite_completions.py", line 375, in _run_command
    result = getattr(plugin, command)(*args, **kwargs)
AttributeError: 'TraditionalBibliographyPlugin' object has no attribute 'on_insert_citation'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "/Users/xiaoweiz/Library/Application Support/Sublime Text 3/Packages/LaTeXTools/latex_cite_completions.py", line 410, in run_plugin_command
    result = _run_command(plugins)
  File "/Users/xiaoweiz/Library/Application Support/Sublime Text 3/Packages/LaTeXTools/latex_cite_completions.py", line 395, in _run_command
    raise BibPluginError(error_message)
LaTeXTools.latex_cite_completions.BibPluginError: TraditionalBibliographyPlugin does not implement `on_insert_citation`
```

**Comment by xiaoweiz** [comment]
Changing the `bibliography` setting to `"new_bibliography"` solves the problem. The issue should be related to the `traditional_bibliography` setting file.

**Comment by ig0774** [comment]
Fixed in v3.7.8.
